CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,4 @@
+Fix: revert usage of restore filter by attribute object ids
 Add missed conf env vars about authentication (#704)
```

Comment:
CNR is supposed to be the delta of changes betwen last published version (in this case 2.7.0) and master. Given the problem was introduced in the current development cycle (i.e. after 2.7.0) I'd suggest to remove this line.

In other words, the bug wasn't in 2.7.0 so no actual fix has been done comparing the current code with 2.7.0.

Reply:
fixed in 71570d41d324369eb5aad2020a764f289d17a7bf